doc/releases/changelog-dev.md

```diff
@@ -141,9 +149,14 @@
 
  <h3>Breaking changes</h3>
 
+* The target wires of the unitary for `ControlledQubitUnitary` are no longer available via `op.hyperparameters["u_wires"]`. 
```

Comment:
Ask @trbromley if we should add a deprecation cycle for this

Reply:
Fairly internal edge case in my mind.